Bellerophon/BellerophonTests/BellerophonTests.swift

```diff
@@ -10,27 +10,127 @@ import XCTest
 @testable import Bellerophon
 
 class BellerophonTests: XCTestCase {
-    
+
+    enum ResponseCases: Int {
+        case KillSwitchOffForceUpdateOff = 0b00
+        case KillSwitchOffForceUpdateOn = 0b01
+        case KillSwitchOnForceUpdateOff = 0b10
+        case KillSwitchOnForceUpdateOn = 0b11
```

Comment:
any reason we're using binary here as opposed to just Int, as specified in the enum declaration?